articles/140_topic_and_service_name_mapping.md

```diff
@@ -76,14 +73,15 @@ The content of substitutions:
 
 - must not be empty
 
+- must be not longer than 256 characters
```

Comment:
To be clear this bullet is talking about "The content of substitutions", which is the stuff inside the `{}` (which is to say not what they get replaced with), i.e. if you have `{foo}` which expands to `bar`, we're talking about the rules for what `foo` can contain.

Based on that, I don't think there's an issue with `/foo/{somethingreallylong...[> 256 characters]...junkjunkjunk}/bar` so long as that substitution is replaced with something that doesn't make the resulting topic invalid. I don't know why you'd do this, but I also don't see why it is a restriction we have to put in place.